src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {EthereumScript, GnosisScript} from 'aave-helpers/ScriptUtils.sol';
+import {AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213} from './AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.sol';
+
+/**
+ * @dev Deploy Gnosis
+ * deploy-command: make deploy-ledger contract=src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol:DeployGnosis chain=gnosis
+ * verify-command: npx catapulta-verify -b broadcast/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol/100/run-latest.json
+ */
+contract DeployGnosis is GnosisScript {
+  function run() external broadcast {
+    // deploy payloads
+    AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213 payload0 = AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213(
+        GovV3Helpers.deployDeterministic(
+          type(AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213).creationCode
+        )
+      );
```

Comment:
```suggestion
    address payload0 =
        GovV3Helpers.deployDeterministic(
          type(AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213).creationCode
        );
```

both is fine, but better align with patched generator if ppl check for reference